test/models/response/CashTransactionSearchResponseSpec.scala

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.response
+
+import play.api.libs.json.{JsSuccess, Json}
+import utils.SpecBase
+import play.api.libs.json._
+
+class CashTransactionSearchResponseSpec extends SpecBase {
+
+  "CashAccountTransactionSearchResponseDetailSpec" must {
+
+    "populate correctly with all the fields" in new Setup {
```

Comment:
Please remove this and tests starts at line 37, 47 as these are inherent functionality. We should only have tests for Json Reads and Writes

Reply:
Could you please remove test at line 27?